ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -22,7 +22,10 @@ def Acosh(*, x, name="Acosh"):
     return ivy.acosh(x)
 
 
-Add = tf_frontend.math.add
```

Comment:
This fix was requried to fix the issue with keyword args as those functions can only accept them. Other alternative is to change the functions inside tf_frontend.math, but I am not sure if it will affect other places

Reply:
I'm sorry, but what was the issue with keyword args? I checked TF docs and `add` arguments are not keyword args 🤔

Reply:
The tests were failing due to that. I am going to mark this one as a draft for now as there are some changes going rn and wait until it gets resolved. I am going to remove that changes later